config/step/ot_finngen_ukb_meta.yaml

```diff
@@ -0,0 +1,13 @@
+defaults:
+  - finngen_ukb_meta
```

Comment:
To some extend this file is not required at all by the dag, you only set here only one parameter

Reply:
Yes, you're right. It's a remnant of how DAGs were organised previously, where this file was required. I'll remove it